liteflow-labs/starter-kit
# Chat page added

### Project organization
- Related [trello card](https://trello.com/c/lYgYDlnW/656-messaging-system-starter-kit-responsive-improvement)

### Description
Removes chat widget from app level and add new chat page

### How to test
Either use navigation after logging in or go to the `/chat` url. Should work both for mobile and desktop.

### Checklist
